### PR TITLE
[REM3-262] serializes the shutdown write and send close flag to ensur…

### PR DESCRIPTION
**Pull request by bmaxwell**
…e the correct right order

https://issues.jboss.org/browse/REM3-262

### CODE REVIEW

File: src/main/java/org/jboss/remoting3/remote/RemoteConnectionHandler.java [hunk 1]
```diff
@@ -418,7 +416,6 @@ public String getRemoteEndpointName() {
 
     protected void closeAction() throws IOException {
         sendCloseRequest();
-        remoteConnection.shutdownWrites();
```
Comment (dmlloyd) [inline src/main/java/org/jboss/remoting3/remote/RemoteConnectionHandler.java]: If we don't shut down writes here (and up above in close()) then can you show that we won't leak connections?

Comment (bmaxwell) [inline src/main/java/org/jboss/remoting3/remote/RemoteConnectionHandler.java]: @elguardian ^^

Comment (elguardian) [inline src/main/java/org/jboss/remoting3/remote/RemoteConnectionHandler.java]: u are right david.

**Comment by bmaxwell** [comment]
Closing PR, as Enrique will update the PR

**Comment by elguardian** [comment]
superseded  https://github.com/jboss-remoting/jboss-remoting/pull/103

**Pull request closed without merging**